src/components/Workspace.jsx

```diff
@@ -175,21 +179,32 @@ export default class Workspace extends React.Component {
     );
   }
 
+  _handleSave() {
+    this.props.onSave();
+    return false;
```

Comment:
Ah yes the false prevents the default browser save action

See https://www.npmjs.com/package/mousetrap

Reply:
OK—I don’t think this is the right method to have that in, then, since a `_handleSave()` method (based on its name) would not naturally know anything about `mousetrap`. I would probably just put it in the handlers themselves e.g.

```js
    const handlers = {
      SAVE: () => {
        this._handleSave();
        return false;
      },
    };
```